daemon/Makefile

```diff
@@ -19,25 +20,31 @@ GO_BINDATA := $(QUIET) go-bindata -prefix ../ -mode 0640 -modtime 1450269211 \
 	-ignore '.+\.o$$' -ignore '.+\.orig$$' -ignore '.+~$$' \
 	-ignore '\.DS_Store'
 
-all: $(TARGET)
+all: $(TARGET) links
+
+links:
+	$(foreach link,$(LINKS), ln -s $(TARGET) $(link);)
 
 clean:
 	@$(ECHO_CLEAN) $(notdir $(shell pwd))
 	$(QUIET)rm -f $(TARGET)
 	$(QUIET)$(GO) clean
+	$(foreach link,$(LINKS), rm $(link);)
 
 ifeq ("$(PKG_BUILD)","")
 
 install:
 	groupadd -f cilium
 	$(INSTALL) -m 0755 -d $(DESTDIR)$(BINDIR)
 	$(INSTALL) -m 0755 $(TARGET) $(DESTDIR)$(BINDIR)
+	$(foreach link,$(LINKS), ln -s $(TARGET) $(DESTDIR)$(BINDIR)/$(link);)
```

Comment:
What is this doing?

Reply:
creating symlinks to cilium-agent binary